_includes/live-demos/premiumskinsandicons-fluent/style.css

```
@@ -0,0 +1,14 @@
+/* Add a border around the editor */
+.my-custom-editor-container {
+    border: 1px solid #CBCBCB;
+    border-top: 0; /* Remove top border because of the dummy header */
+}
+
+.dummy-header {
+    background-color: {{site.primaryColor}};
+    color: #fff;
```

For consistency with the other colour
```suggestion
    color: #FFF;
```